packages/origin-247-certificate/src/evented-certificate/repositories/CertificateEvent/CertificateEvent.entity.ts

```diff
@@ -0,0 +1,32 @@
+import { CreateDateColumn, Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+import { IIssueCommand, ITransferCommand, IClaimCommand } from '../../../types';
+import { CertificateEventType } from '../../events/Certificate.events';
+
+export const tableName = 'certificate_event';
+
+@Entity(tableName)
+export class CertificateEventEntity {
```

Comment:
We should think about indexes here. I think indexing by `internalCertificateId` will be necessary